holopy/scattering/tests/test_interface.py

```diff
@@ -216,10 +216,34 @@ def test_interpret_specified_theory(self):
 
 
 class TestValidateScatterer(unittest.TestCase):
+    priors = [prior.Uniform(1, 2), prior.Gaussian(1, 0.1),
+              prior.BoundedGaussian(1, 0.1, 0, 3),
+              prior.TransformedPrior(np.sqrt, prior.Uniform(1, 4)),
+              prior.ComplexPrior(prior.Uniform(1, 2), 0.001)]
+
+    @attr('fast')
+    def test_all_priors_accounted_for(self):
```

Comment:
It might be good to add a comment here clarifying that this is a test of the tests, rather than a test of the code. Or perhaps rename this as `test_that_the_tests_account_for_all_priors` or something.